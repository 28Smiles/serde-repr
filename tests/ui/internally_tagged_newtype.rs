use serde_repr::Serialize_repr;

#[derive(Serialize_repr)]
#[repr(u8)]
#[serde(tag = "type")]
enum SmallPrime {
    Two(u8),
    Three(u8),
    Five(u8),
    Seven(u8),
}

fn main() {}
