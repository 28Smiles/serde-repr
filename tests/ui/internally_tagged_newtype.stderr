error: Internally tagged enums with unnamed fields are not supported
 --> tests/ui/internally_tagged_newtype.rs:3:10
  |
3 | #[derive(Serialize_repr)]
  |          ^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `Serialize_repr` (in Nightly builds, run with -Z macro-backtrace for more info)
